docs/release/trg-1/trg-1-2.md

```diff
@@ -2,15 +2,74 @@
 title: TRG 1.02 - INSTALL.md
 ---
 
-| Status | Created      | Post-History |
-|--------|--------------|--------------|
-| Active | 07-Mar-2023  |              |
-| Draft  | 13-Sept-2022 |              |
+| Status | Created      | Post-History                                        |
+|--------|--------------|-----------------------------------------------------|
+| Update | 13-Dez-2023  | add best practices and an example, rephrase the why |
+| Active | 07-Mar-2023  |                                                     |
+| Draft  | 13-Sept-2022 |                                                     |
 
 ## Why
 
-If a repository contains code which needs to be configured/setup or installed, the `README.md` file should cover this information. If this is too much text, it can be moved into a `INSTALL.md` file.
+If a repository contains code which needs to be configured/setup or installed, the `README.md` file should cover this information.
+
+If these instructions are too long, they can be moved into a dedicated `INSTALL.md` file.
+
+Having a proper "install.md" file is crucial for ensuring a seamless and user-friendly experience for the product-users.
+
+In conclusion, a proper install.md file tailored for end-users, but also developers. It is not a technical necessity, it's a strategic investment in user satisfaction, community engagement, and the overall success of the product. It reflects a commitment to inclusivity and usability, ensuring that a diverse user base can effortlessly install and enjoy the benefits of your product.
 
 ## Description
 
 Create a `INSTALL.md` file in the main folder of the repository and add all installation relevant information.
+
+## Best Practices
+
+- add a clear installing guide through our official helm chart repo
+- add table of content of your installation guide
+- add a numeric step-by-step installation guide with command copy options
+- add prerequisites section for the installation
+- add a reference your helm chart README.md file
+- add guidance on how to verify the installation
+- add local deployment advisories
+- add a troubleshooting section
+- add a FAQ section
+- add additional resources information
+- add uninstallation guide
+- add upgrade guide
+- add security considerations
+- add contact information for support or feedback
+- add a thankful note for the user for using the product
```

Comment:
Just a suggestion, it may look cleaner removing 'add' from each line, should be enough to state somewhere above to include following sections.

Reply:
[eb1f28d](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/559/commits/eb1f28dc0ff569251dc8c340b33e7ffa4db250d6)